hugo/content/blog/2022-01-16-mumble-1-4-230.md

```diff
@@ -0,0 +1,392 @@
+size needed to display the respective information. Thus it usually does not take up a lot of space and can therefore be left floating in the corner of
+the screen without consuming too much screen real estate.
+
+Note that the TalkingUI is highly configurable (Settings -> User Interface) and also provides selection and context-menu support just as you get
+it in the main window. In fact, the selection between TalkingUI and main window is synchronized and thus this plays really nice with the new "whisper
+to selected" feature allowing to whisper to whatever currently is selected in the UI (be it a channel or a user).
+
+![TalkingUI](/blog/media/TalkingUI.png)
+
+
+### UI indication for access restrictions
+
+In previous versions of Mumble there was no indication of whether you are able to join a channel or not. You just had to try it out and either it
+worked or you got a permission denied message. This feature solves this problem by marking "access-restricted" channels with a lock icon. Access
+restricted channels are all channels that are either only open to a specific group on the server or require a password (access token) to join.
+
+In total, a channel can be in one of three states: either it is not access-restricted in which case it just looks like channels used to look prior to
+the introduction of this feature or it is indeed access-restricted. In the latter case a lock icon is shown to the right of channel name. This look
+can either be open and green or closed and red. Green means that you can enter the channel nonetheless (you belong to the required group or have the
+correct password (access token) set) whereas red means that you are not allowed to enter the channel.
+
+Note that this is also a feature that requires server-side support and thus only takes effect on servers that are at least running Mumble version
+1.4.
+
+![Access restricted channels](/blog/media/AccessRestrictedChannels.png)
+
+
+### Markdown
+
+For those of you who don't know what markdown is, we recommend a quick read of the respective [Wikipedia
+article](https://en.wikipedia.org/wiki/Markdown). In simple words it allows formatting of text by simply typing a few extra characters in it.
+
+Mumble now supports markdown for text messages written in the normal chat bar. It is not supported for cases in which you use the dedicated message
+editor (e.g. for private messages). Furthermore only a subset of markdown is supported: sections, inline-code, code-blocks, bold, italic and
+strike-through text, links and quotes. _Not_ supported are lists and tables.
+
+This feature is probably best explained by an example. Try pasting the following text into the chat bar and see for yourself:
+
+````
+# Markdown
+## Code
+We can have `inline` code and
+```
+code
+    blocks
+```
+
+## Emphasis
+**bold** and *italic* (__bold__ and _italic_ also works) text is supported. ~~Scratch that~~
+
+## Links
+We can have [links](google.de) and www.google.de
```

Comment:
We could use a neutral website; our own website.
```suggestion
We can have [labeled links](https://www.mumble.info) and www.mumble.info
```

Reply:
But then the screenshot doesn't match anymore - and in the end I don't think it really matters, does it? I just used the same example that I used in my original PR :shrug: